BUILDIT.md

```diff
@@ -1546,27 +1546,21 @@ defmodule AppWeb.AppLive do
     end
   end
 
-  # assign default values to socket:
-  defp assign_socket(socket) do
```

Comment:
I'm not using this function anymore as the items were loaded twice once in mount and then in the `handle_params` function. I'm just loading them in the `handle_params` function now.
The `active: %Item{}` also wasn't used and the `editing: nul` value was removing current edits when an event (toggle, add, edit...) was triggered by another client.